client/src/components/route/PrivateRoute.js

```diff
@@ -47,8 +45,6 @@ const PrivateRoute = () => {
           />
           <DashboardLayoutRoute path="/team" exact component={TeamPage} />
```

Comment:
Since we get rid of the EditorLayoutRoute component, can we also get rid of the DashboardLayoutRoute component and simply use <Route path="/team" exact component={TeamPage} /> like what we have in the PublicRoute component?

Reply:
I would say keeping `DashboardLayoutRoute`, because it actually has the `header` and `sidebar` in it, if we take the page content out, we'll have to put `header` and `sidebar` in `PrivateRoute.js`, which I personally think that is a bit messy when looking into `PrivateRoute.js`.  What do you think?

Reply:
True. Make sense to me.